src/boot/ostree-remount.service

```diff
@@ -17,7 +17,6 @@
 Description=OSTree Remount OS/ Bind Mounts
 Documentation=man:ostree(1)
 DefaultDependencies=no
-ConditionKernelCommandLine=ostree
```

Comment:
This should change to `ConditionPathExists=/run/ostree` instead; we don't want to run it unconditionally.

That's true of basically everything that is `After=ostree-prepare-root.service` which creates it.

That change can land now, independent of this change.  Also, we can start porting things in [in fedora-coreos-config](https://github.com/search?q=repo%3Acoreos%2Ffedora-coreos-config%20ConditionKernelCommandLine%3Dostree&type=code) etc.

Reply:
This file is no longer changed